src/components/calcite-input/calcite-input.tsx

```diff
@@ -446,6 +450,7 @@ export class CalciteInput {
 
   private clearInputValue = () => {
     this.value = "";
+    this.emitInputInput();
```

Comment:
could you just `@Watch()` the `value` prop and then emit?

Reply:
This is emitted any time the value is cleared or typed. Is that all times `value` is modified? If so, then a `@Watch` makes sense.